src/main/java/ita/softserve/course_evaluation_analytics/repository/UsersRepository.java

```diff
@@ -10,4 +10,6 @@
 
     List<Role> findRoleByUserEmail(String email);
     Optional<Users> findByEmail(String email);
+    Users findTeacherByCourseId(Long id);
```

Comment:
Why ```findTeacherByCourseId``` return ```User```? Should be a teacher, isn't it?

Reply:
I deleted this method